src/marp.ts

```diff
@@ -25,6 +26,7 @@ export interface MarpOptions extends Options {
           | { [attr: string]: boolean | ((value: string) => string) }
       }
   markdown?: object
+  mathjax?: boolean
```

Comment:
I think that interface of constructor option is not intuitive. `math: 'mathjax'` would be better than an individual option.

Reply:
Nice idea. Let me confirm you.

```
math: 'katex' # enable katex as same as `true`
math: true # enable katex for backward compatibility
math: { ... } # enable katex with options
math: 'mathjax' # enable mathjax
```

Do you want to do like this?